calculator.js

```diff
@@ -1,2 +1,81 @@
-var calculator = {
+
+ //var num1 = prompt("please enter first numbers");
+
+ //var num2 = prompt("please enter second numbers");
+
+console.log("The function name are addition, subtract, multiply, devide,power, and sqrt")
+
+ //var calculator = function addition("a","b","return (a + b)");
+//var myFunction = function add(a, b) {return a * b};
+var calculator={
+  addition: function addition(a, b)
+  {
+    return (a + b);
+  },
+  subtract: function subtract(a, b)
+  {
+   return (a - b);
+ },
+  multiply: function multiply(a, b)
+  {
+    return (a * b);
+  },
+  devide: function devide(a, b)
+  {
+    return (a / b);
+  },
+  power: function power(a,b)
+  {
+    return (Math.pow(a, b));
```

Comment:
You can also use the `a**b` syntax instead of `Math.pow(a,b)`. Use whichever one you like more!